tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ElasticSearchViewsSpec.scala

```diff
@@ -421,5 +421,19 @@ class ElasticSearchViewsSpec extends BaseSpec with EitherValuable with CirceEq {
       }
     }
 
+    "return the view's mapping" in {
```

Comment:
We need other tests with a user with a missing permission and for a non-existing view